src/RichText.js

```diff
@@ -0,0 +1,82 @@
+import {Editor, ContentState, convertFromHTML, DefaultDraftBlockRenderMap, DefaultDraftInlineStyle, CompositeDecorator} from 'draft-js';
+const ReactDOM = require('react-dom');
+var sdk = require('./index');
+
+const BLOCK_RENDER_MAP = DefaultDraftBlockRenderMap.set('unstyled', {
+    element: 'p' // draft uses <div> by default which we don't really like
+});
+
+const styles = {
+    BOLD: 'strong',
+    CODE: 'code',
+    ITALIC: 'em',
+    STRIKETHROUGH: 's',
+    UNDERLINE: 'u'
+};
+
+export function contentStateToHTML(contentState:ContentState): String {
+    const elem = contentState.getBlockMap().map((block) => {
+        const elem = BLOCK_RENDER_MAP.get(block.getType()).element;
+        const content = [];
```

Comment:
I'm inclined to agree with kegan, mostly because of his first point. I'd like to reserve `const` for places where we really are declaring constants.

(I don't set much store by the confusion with other languages: every language has its own foibles in this respect, and javascript's handling of `const` is comparable to that of Java/C/etc)